deployment/conf_files/etc/init/vnet-vnmgr.conf

```diff
@@ -8,6 +8,10 @@ respawn limit 5 60
 env NAME=vnmgr
 
 script
+    ## log
+    #exec 2>>/var/log/openvnet/vnmgr-upstart.log
+    #set -x
```

Comment:
it's a good way to keep this piece code for debugging purposes, so you don't need to spend time trying to find out how to enable logging the upstart script again it's only 3 lines